packages/editor/src/pages/EditorPage.tsx

```diff
@@ -60,7 +61,7 @@ export const EditorPage = () => {
   const [params] = useSearchParams()
   const { scenePath, projectName } = useHookstate(getMutableState(EditorState))
   const supportedBrowser = useHookstate(isSupportedBrowser)
-  const acknowledgedUnsupportedBrowser = useHookstate(false)
+  const acknowledgedUnsupportedBrowser = useHookstate(localStorage.getItem('acknowledgedUnsupportedBrowser') ?? false)
```

Comment:
can you do it using state definition? (using `syncStateWithLocalStorage`)
 (example: https://github.com/ir-engine/ir-engine/blob/641da655a0365ddb28c9f9a288a54ffd6948cf1c/packages/editor/src/services/HierarchyNodeState.ts/#L42)